.circleci/config.yml

```diff
@@ -266,10 +238,8 @@ jobs:
       - teardown_vm
   'End-to-End Tests':
     executor:
-      name: amphtml-large-executor
+      name: amphtml-xlarge-executor
```

Comment:
I was going to suggest unblocking this PR by switching back to a linux machine just for the E2E tests and dealing with it separately. I then realized it's not only this job, but also the "Experiment A/B/C Tests" jobs that run E2E tests, so they'd all need to switch back to machine from docker.

May still be worth doing to unblock all the other improvements in this PR. No strong preference from me, so I'll leave this choice to you and @rileyajones.

Reply:
I'm still trying things out, but I'm leaning towards keeping this `large` (there is no measurable improvement with `xlarge`) and disabling the dozen tests that became flaky. I'll re-run the workflow a few times to make sure this works as expected

Reply:
btw, I tried switching to a machine, but it looks like there's some incompatibilities when moving the data between the Docker instances and the VM instances. Nothing that can't be resolved with enough hammers and nails, but I figured since we want to end up with insta-loading Docker containers anyways, I'd rather unblock this PR by skipping flakes rather than making a complicated startup script

Reply:
Good data point, and SGTM. 👍

Reply:
This makes sense to me, the performance improvements are quite compelling.

A lot of the tests that are flaking with Docker appear to be commonly flaky tests anyway so skipping them might provide a good reason to look more closely at them individually later.